src/agent/tools/multi_doc_synthesizer.py

```
@@ -0,0 +1,262 @@
+    **Best practices:**
+    - Use 2-5 documents for best results (10 max)
+    - Provide specific synthesis_query (not generic)
+    - Use compare mode for differences, summarize for unified overview
+    - Results cite all source documents
+
+    **Method:** filtered_search per document + LLM synthesis
+    
+    **Cost:** Higher (multiple retrievals + LLM synthesis)
+    """
+
+    input_schema = MultiDocSynthesizerInput
```

Ruff (0.14.5)

Undefined name `MultiDocSynthesizerInput`

(F821)